azext_iot/_params.py

```diff
@@ -378,3 +385,70 @@ def load_arguments(self, _):
+        context.argument('interface', options_list=['--interface', '-i'],
+                         help='Target interface name. This should be the name of the interface not the urn-id.')
+        context.argument('command_name', options_list=['--command-name', '--cn'],
+                         help='IoT Plug and Play interface command name.')
+        context.argument('command_payload', options_list=['--command-payload', '--cp', '--cv'],
+                         help='IoT Plug and Play interface command payload. '
+                         'Content can be directly input or extracted from a file path.')
+        context.argument('interface_payload', options_list=['--interface-payload', '--ip', '--iv'],
+                         help='IoT Plug and Play interface payload. '
+                         'Content can be directly input or extracted from a file path.')
+        context.argument('schema', options_list=['--schema', '-s'],
+                         arg_type=get_three_state_flag(),
+                         help='Switch to skip the operation and retrieve the PnP repository '
+                         'interface entity schema. Default: false')
+
+    with self.argument_context('iot dt monitor-events') as context:
+        context.argument('repair', options_list=['--repair'],
+                         arg_type=get_three_state_flag(),
+                         help='Reinstall uamqp dependency compatible with extension version.')
+        context.argument('consumer_group', options_list=['--consumer-group', '--cg'],
+                         help='Specify the consumer group to use when connecting to event hub endpoint.')
+        context.argument('properties', options_list=['--properties', '--props', '-p'], arg_type=event_msg_prop_type)
+
+    with self.argument_context('iot pnp') as context:
+        context.argument('model', options_list=['--model', '-m'],
+                         help='Target capability-model urn-id. Example: urn:example:capabilityModels:Mxchip:1')
+        context.argument('interface', options_list=['--interface', '-i'],
+                         help='Target interface urn-id. Example: urn:example:interfaces:MXChip:1')
+
+    with self.argument_context('iot pnp interface') as context:
+        context.argument('interface_definition', options_list=['--definition', '--def'],
+                         help='IoT Plug and Play interface definition written in PPDL (JSON-LD). '
+                         'Can be directly input or a file path where the content is extracted.')
+
+    with self.argument_context('iot pnp interface list') as context:
+        context.argument('search_string', options_list=['--search', '--ss'],
+                         help='Searches IoT Plug and Play interfaces for given string in the'
+                              ' \"Description, DisplayName, comment and Id\".')
+        context.argument('page_size', options_list=['--page', '--ps'],
```

Comment:
Does it make sense that the end user needs to think about `page_size` on a list operation? Unless this is intended in behavior to be like `top` in our other list operations (then it should be called `top`)? If its really an API page size, as in chunks of elements to pull at a time to collect the whole collection, then it would be better to hide from the user.

Reply:
Point is it looks like an implementation detail.

Reply:
replace "top" in place of page_size and token